memorywipe.py

```diff
@@ -362,15 +363,15 @@ def _select_enhance(self):
                     "This mode writes predetermined data patterns set by the manufacturer to all areas including bad blocks")
                 click.echo("Please wait... this may take a long time. At-least:")
                 hdparm = subprocess.run(["sudo", "hdparm", "-I", f"{self.partition}"], capture_output=True)
-                subprocess.check_output(["grep", "-i", "erase unit"], input=hdparm.stdout)
```

Comment:
I don't know the reason, but here and also in line 373, if check_output is present, the command is not working. Replacing it with run is working.

Reply:
I have fixed this and other similar commands. Interchanged the run() and check_output() function calls in these lines.